### DISCUSSION TITLE
How to use custom columns?

### DISCUSSION THREAD

**Discussion by otbutz** [F43nd1r/Acrarium#387] [Q&A] [answered]
I couldn't find any documentation how to use this feature.

**Comment by F43nd1r** [comment, marked as answer]
Yeah I guess [this](https://github.com/F43nd1r/Acrarium/commit/9dca1ea656a07c26aa0c96c6d58bec7488ff0b4f#commitcomment-50317195) is not the best place for documentation. Feel free to add something to the wiki when you have time.
